sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContextBase.java

```diff
@@ -132,12 +137,29 @@ protected void initialize(String collectionRid,
                     String.format("Could not find partition key range for continuation token: {0}", needle));
         }
 
+        List<PartitionKeyRange> replacementRanges;
+
+        // find what ranges make up the supplied continuation token
+        replacementRanges =
+            partitionKeyRanges.stream().filter(p -> range.getMin().compareTo(p.getMinInclusive()) <= 0 &&
```

Comment:
please put each observable operation on it's own line.

Reply:
Done